tests/manual/desktopViewTestPlan.md

```diff
@@ -0,0 +1,100 @@
+
+#### Method 1: Select a city from the auto-completed search input box
+
+1. Use keyboard or mouse to focus in the search input box;
+2. Start to type a letter for example "t";
+3. A drop down list box shows up with all matching city names starting with "t". See the screenshot below;
+4. Select the city you are interested in from the auto-completed drop down list box.
+
+![Desktop View - Auto Complete Search](images/searchAutoComplete.png)
+
+#### Method 2: Select a postal code from the auto-completed search input box
+
+1. Use keyboard or mouse to focus in the search input box;
+2. Start to type a letter then a number for example "m9";
+3. A drop down list box shows up with all matching postal codes starting with "m9". See below;
+4. Select the postal code you are interested in from the auto-completed drop down list box.
+
+#### Method 3: Select a city from the cities list
+
+1. Search a city from the cities list:
+
+### Verify search results page
+
+All 3 methods above will end up as the screenshot below:
+
+![Desktop View - Search Results](images/searchResults.png)
+
+1. The cities list is replaced by search results;
+2. The map zooms in to the selected city;
+3. An "X" button is shown at the right end of the search input box.
+
+### Reset search
+
+1. Click the "X" button at the right end of the search input box;
+2. Everything restores to the initial load state: search input box is empty; the "X" button at the right end of the
+search box disappears; the cities list is back, the map restores to the overview of Ontario.
+
+## Results page
+
+### Select a COVID assessment centre from the search results page
+
+1. Select a city from the cities list;
+2. The results page is shown as the screenshot above;
+3. Scroll up and down on the results page, find and select a COVID assessment centre from the search results page;
+4. The corresponding hospital description is shown at the bottom of the page. See the screenshot below:
+
+![Desktop View - Hospital Description](images/hospitalDescription.png)
+
+### Back to locations
+
+1. Click "Back to locations" button on the search results page;
+2. Everything restores to the initial load state: search input box is empty; the "X" button at the right end of the
+search box disappears; the cities list is back, the map restores to the overview of Ontario.
+
+## Accessibility filter panel
+
+### Reset filters
+
+1. Select a number of checkboxes in this panel. While checking, the number at the top right corner of the panel
+increases to show the total number of selected filters;
+2. Deselect some checkboxes, the number at the top right corner of the panel decreases;
+3. Make sure some checkboxes are checked;
+4. Click "reset filters" button;
+5. All checkboxes are unchecked. the number at the top right corner of the panel restores to 0.
+
+### Apply filters
+
+This feature is not yet implemented. It will be implemented when the accessibility data for COVID assessment centres
```

Comment:
If this is merged up with main, the feature is implemented with respect to the placeholder data